caffe2/python/tutorials/py_gen/MNIST.py

```diff
@@ -318,25 +332,23 @@ def AddBookkeepingOperators(model):
     test_model, batch_size=100,
     db=os.path.join(data_folder, 'mnist-test-nchw-lmdb'),
     db_type='lmdb')
-softmax = AddLeNetModel(test_model, data)
+softmax = AddModel(test_model, data)
 AddAccuracy(test_model, softmax, label)
 
-# Deployment model. We simply need the main LeNetModel part.
+# Deployment model. We simply need the main AddModel part.
 deploy_model = model_helper.ModelHelper(
     name="mnist_deploy", arg_scope=arg_scope, init_params=False)
-AddLeNetModel(deploy_model, "data")
+AddModel(deploy_model, "data")
 # You may wonder what happens with the param_init_net part of the deploy_model.
 # No, we will not use them, since during deployment time we will not randomly
 # initialize the parameters, but load the parameters from the db.
 
 
-# Now, let's take a look what the training and deploy models look like, using the simple graph visualization tool that Caffe2 has. If the following command fails for you, it might be because that the machine you run on does not have graphviz installed. You can usually install that by:
-# 
-# ```sudo yum install graphviz```
+# Now, let's take a look what the training and deploy models look like, using the simple graph visualization tool that Caffe2 has. If the following command fails for you, it might be because that the machine you run on does not have graphviz installed. Which you need to install through a package manager of your choice. 
```

Comment:
Now, let's take a look what the training and deploy models look like using the simple graph visualization tool that Caffe2 has. If the following command fails for you, it might be because your machine does not have graphviz installed. You'll need to install it through the package manager of your choice.